cluster/kubevirtci.sh

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17.0'}
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17'}
```

Comment:
You have to fix this in https://github.com/kubevirt/cluster-network-addons-operator/blob/master/automation/check-patch.e2e-workflow-k8s.sh and https://github.com/kubevirt/cluster-network-addons-operator/blob/master/automation/check-patch.e2e-lifecycle-k8s.sh too